query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverter.java

```diff
@@ -47,6 +48,8 @@ public String convert(
         return this.functionToStringForDistinctCount(function, argumentConverter);
       case QUERY_FUNCTION_CONCAT:
         return this.functionToString(this.toPinotConcat(function), argumentConverter);
+      case PINOT_DISTINCT_COUNT_MV_FUNCTION:
```

Comment:
nit: this constant can go to `QueryFunctionConstants` class, similar to QUERY_FUNCTION_DISTINCTCOUNT